File: Fundamentals of C programming/step07/7.1/7.1.05.c
/*

Факториал.
Для целого числа k(0≤k≤12)k(0≤k≤12) посчитать k!k!.

Входные данные:
Одно целое число kk, (0≤k≤12)(0≤k≤12).

Выходные данные:
Значение факториала числа kk.

Sample Input:

4

Sample Output:

24

*/

#include <stdio.h>

int main(void) {

  int k,f=1;
  scanf("%d",&k);
  
  for (int i = 1; i <= k; i++){
	f = f*i;
  }
  printf("%d\n",f);

  return 0;
}

File: Programming: Python/step07/7.01/7.1.05.py
def check_number(k):
    if 24 <= k <= 25:
        print("ПОПАЛ")
    elif k >= 26:
        print("ПЕРЕЛЕТ")
    elif 0 < k < 24:
        print("НЕДОЛЕТ")
    else:
        print("МАЗИЛА")

k = int(input())
check_number(k)
